test/moves/glaive_rush.test.ts

```diff
@@ -62,7 +62,7 @@ describe("Moves - Glaive Rush", () => {
   });
 
   it("interacts properly with multi-lens", async () => {
-    game.override.startingHeldItems([{ name: "MULTI_LENS", count: 2 }]).enemyMoveset([MoveId.AVALANCHE]);
+    game.override.enemyAbility(Abilities.PARENTAL_BOND).enemyMoveset([MoveId.AVALANCHE]);
```

Comment:
```suggestion
    game.override.ability(Abilities.PARENTAL_BOND).enemyMoveset([MoveId.AVALANCHE]);
```
(The multi lens was originally given to the player, not the enemy)